From ff4b6cab1dead31bad114321c211797ac28b36b1 Mon Sep 17 00:00:00 2001
From: Jayaprakash Mutyala <mutyalax.jayaprakash@intel.com>
Date: Wed, 10 Nov 2021 15:17:13 +0000
Subject: [PATCH] Add HSBP FRU details in json configuration

Added Intel specific HSBP CPLD FRU details in json configuration file.

Tested:
1. Detected and read the HSBP FRU with new address on reworked board.
2. busctl tree xyz.openbmc_project.HsbpManager
└─/xyz
  └─/xyz/openbmc_project
    ├─/xyz/openbmc_project/inventory
    │ └─/xyz/openbmc_project/inventory/item
    │   ├─/xyz/openbmc_project/inventory/item/drive
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_1
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_10
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_11
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_12
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_13
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_14
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_15
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_16
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_2
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_3
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_4
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_5
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_6
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_7
    │   │ ├─/xyz/openbmc_project/inventory/item/drive/Drive_8
    │   │ └─/xyz/openbmc_project/inventory/item/drive/Drive_9
    │   └─/xyz/openbmc_project/inventory/item/hsbp
    │     ├─/xyz/openbmc_project/inventory/item/hsbp/HSBP_1
    │     └─/xyz/openbmc_project/inventory/item/hsbp/HSBP_2
    └─/xyz/openbmc_project/software
      ├─/xyz/openbmc_project/software/HSBP_1
      └─/xyz/openbmc_project/software/HSBP_2

Signed-off-by: Jayaprakash Mutyala <mutyalax.jayaprakash@intel.com>
---
 configurations/F2U8X25 HSBP.json | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/configurations/F2U8X25 HSBP.json b/configurations/F2U8X25 HSBP.json
index 60e7817..cdf9076 100644
--- a/configurations/F2U8X25 HSBP.json	
+++ b/configurations/F2U8X25 HSBP.json	
@@ -7,6 +7,13 @@
                 "Name": "F2U8X25 HSBP1 FRU",
                 "Type": "EEPROM"
             },
+            {
+                "Address": "0x68",
+                "Bus": "$bus",
+                "Index": 1,
+                "Name": "HSBP 1",
+                "Type": "Intel HSBP CPLD"
+            },
             {
                 "Address": "0x70",
                 "Bus": "$bus",
@@ -84,6 +91,13 @@
                 "Name": "F2U8X25 HSBP2 FRU",
                 "Type": "EEPROM"
             },
+            {
+                "Address": "0x69",
+                "Bus": "$bus",
+                "Index": 2,
+                "Name": "HSBP 2",
+                "Type": "Intel HSBP CPLD"
+            },
             {
                 "Address": "0x73",
                 "Bus": "$bus",
-- 
2.17.1

